src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
+  const response = await fetch(`${BASE_URL}/users`);
+
```

Comment:
Consider adding error handling to this function, especially for better resilience in case of API failures